src/apps/chifra/pkg/types/types_namedblock.go

```diff
@@ -11,24 +11,37 @@ package types
 // EXISTING_CODE
 // EXISTING_CODE
 
-type RawNamedBlock interface{}
+type RawNamedBlock struct {
+	BlockNumber string `json:"blockNumber"`
+	Timestamp   string `json:"timestamp"`
+	Date        string `json:"date"`
+	Name        string `json:"name"`
+}
 
 type SimpleNamedBlock struct {
 	BlockNumber uint64 `json:"blockNumber"`
-	Timestamp uint64 `json:"timestamp"`
-	Date string `json:"date"`
-	Name string `json:"name,omitempty"`
+	Timestamp   int64  `json:"timestamp"`
```

Comment:
This is also same as the Raw type. Maybe we don't need both?

Reply:
Avoiding special cases in auto generated code.